### ISSUE TITLE
Provide visible, keyboard-navigable button to show shortcuts help modal dialog

### ISSUE THREAD

**Issue by cpcallen** [rachel-fenichel/blockly-keyboard-experimentation#100]
It should be possible for developers to add a visible button to their app that opens the shortcuts help modal, and which can be accessed via keyboard navigation.  This plugin's test page should include such a button.

Possible approaches:

* Document API to call (from a normal form button outside the workspace)
* Provide an option that creates an on-workspace button in the style of the zoom controls.
  * (Can we even navigate to such controls at the moment?)

**Comment by cpcallen** [comment]
Per discussion with Micro:bit team: maybe this should be a general help button, not just a keyboard shortcuts help button.

**Comment by bmxedd** [comment]
The UI could be provided in a number ways:

- Always open the shortcut dialog, provide a link to external help within the dialog.
  - Shortcuts may not always be what the user wanted.
- Just open the help page in a new tab, shortcuts are a section within the help. 
  - Multiple clicks to get shortcuts, if you just wanted a quick reference.
  - We also heard users wanted to have shortcuts as a reference side by side to their workspace. If the shortcuts were on an different page, this would enable that.
- Provide a menu which allows choice of shortcuts and direct help.
  - This can scale to having other items like sending feedback. 
  - Example from gmail:

![Image](https://github.com/user-attachments/assets/7a381ae4-443d-4a8b-bb46-cacabad9bb2a)

**Comment by maribethb** [comment]
This is the host application's responsibility, not Blockly's
